deployments/helm/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: beyla
-version: 0.1.0
-appVersion: 1.2.0 
+version: 0.2.0
+appVersion: 1.4.1
```

Comment:
We'll have to update this to 1.5.0 to really make this chart work, once we release Beyla 1.5.0